db/seeds.rb

```diff
@@ -49,76 +49,104 @@
     full_name: "Josh Cheek",
     email: "demo+josh@jumpstartlab.com" }])
 
-Breed.create([
-  { name: "sabertooth",
-    retired: false,
-    description: "A tasty large fanged feline once thought to be extinct. Don't worry, we'll get the job done." },
-
-  { name: "prionailurus viverrinus",
-    retired: false,
-      description: "This cat lives near marshes, mangroves, rivers and streams in India and Southeast Asia. Our finest import."},
-
-  { name: "oncilla",
-    retired: false,
-    description: "This small spotted cat's meat is lean and tender. Goes great between two slices of our signature Rye bread." },
-
-  { name: "oriental shorthair",
-    retired: false,
-    description: "Like the Siamese, Oriental Shorthairs have almond-shaped eyes, a triangular head shape, large ears, and an elongated, slender, and muscular body. Excellent for an appetizer." },
-
-  { name: "black cat",
-    retired: false,
-    description: "Halloween is always just around the corner. Fill up on one of these cheap tickets and we'll let you keep the... remains." }])
-
-Breed.find_by(name: "sabertooth").cats.create([
-  { name:"sabertooth 1",
-    price: 1000,
-    retired: false,
-    description: "cool cat 1" },
-
-    { name:"sabertooth 2",
-      price: 2000,
-      retired: false,
-      description: "cool cat 2" }])
-
-Breed.find_by(name: "oncilla").cats.create([
-  { name:"oncilla 1",
-    price: 2000,
-    retired: false,
-    description: "cool cat 3" },
-
-    { name:"oncilla 2",
-      price: 3000,
-      retired: false,
-      description: "cool cat 4" }
-])
-
-Breed.find_by(name: "oriental shorthair").cats.create([
-  { name:"oriental shorthair 1",
-    price: 4000,
-    retired: false,
-    description: "cool cat 5" },
-
-    { name:"oriental shorthair 2",
-      price: 5000,
-      retired: false,
-      description: "cool cat 6" }
-])
-
-Breed.find_by(name: "black cat").cats.create([
-  { name:"black cat 1",
-    price: 3500,
-    retired: false,
-    description: "cool cat 7" },
-
-    { name:"black cat 2",
-      price: 4000,
-      retired: false,
-      description: "cool cat 8" }
-])
-
-Breed.find_by(name: "prionailurus viverrinus").cats.create(
-  { name:"fishing cat 1",
-    price: 1000,
-    retired: false,
-    description: "cool cat 9" })
+
+  def users_minus_dj
+    users_minus_dj = User.all - [User.first]
+  end
+
+  users_minus_dj.each do |user|
+    5.times do |iteration|
```

Comment:
Unused block argument - `iteration`. You can omit the argument if you don't care about it.